docs/CHANGELOG.md

```diff
@@ -13,12 +13,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Matic autoconfig is now enabled for mainnet. Matic nops should remove any custom tweaks they have been running with. In addition, we have better default configs for Optimism, Arbitrum and RSK.
 
-- It is no longer required to set `DEFAULT_HTTP_ALLOW_UNRESTRICTED_NETWORK_ACCESS=true` to enable local fetches on bridge tasks. Please remove this if you had it set and no longer need it, since it introduces a slight security risk.
+- It is no longer required to set `DEFAULT_HTTP_ALLOW_UNRESTRICTED_NETWORK_ACCESS=true` to enable local fetches on bridge or http tasks. If the URL for the http task is specified as a variable, then this variable is still needed. Please remove this if you had it set and no longer need it, since it introduces a slight security risk.
```

Comment:
I would recommend instead that people set it on a per-job basis rather than using the global env.

We may also want to rename the env to `DANGEROUSLY_` or `UNSAFE_` to discourage usage.

Reply:
If we did that we'd have to support two variables for the same thing, and it'd be unclear for which version they'd apply. I'd prefer to leave this as is for now and not break backwards compatibility and just recommend the per job basis instead.

Reply:
Updated to mention task level variable.